src/Modules/quorum.ts

```diff
@@ -1,6 +1,42 @@
-import { GraphData } from "../GraphTypes";
+import { GraphData, GraphNode } from "../Types/GraphTypes";
+import { NetworkGraphNode } from "../Types/NetworkTypes";
 import dummydata from "../dummydata";
 
+const networkData = dummydata as { nodes: NetworkGraphNode[] };
+
+function convertResponseToGraphData(json: {
+  nodes: NetworkGraphNode[];
+}): GraphData {
+  const convertedNodes: GraphNode[] = json.nodes.map(gn =>
+    Object.assign({}, gn, { id: gn.node })
+  );
+
+  const data: GraphData = {
+    nodes: convertedNodes,
+    links: []
+  };
+  json.nodes.forEach(n => {
+    let targets: string[] = [];
+    let others = n.qset.v;
```

Comment:
I think the usage of `others` / `other` as var names might be a bit confusing – could we use a more specific name? Something like `nonTargetNodes`, or something else that you think makes more sense?